test/e2e/storage/volumes.go

```diff
@@ -686,4 +686,54 @@ var _ = framework.KubeDescribe("Volumes [Volume]", func() {
 			framework.TestVolumeClient(cs, config, &fsGroup, tests)
 		})
 	})
+	////////////////////////////////////////////////////////////////////////
+	// Azure Disk
+	////////////////////////////////////////////////////////////////////////
+	framework.KubeDescribe("Azure Disk", func() {
```

Comment:
Do you need "[Feature:Volumes]" in the text? AFAIK, this causes the test to be run in a separate suite and indicates that the test has non-default requirements to run. I think not but just checking.